libs/testing/containers/src/lib/testing-containers.ts

```diff
@@ -44,3 +46,23 @@ export const startRedisCluster = async () => {
 export const stopRedis = () => {
   redisClusterContainer.stop()
 }
+
+export const startLocalstack = async () => {
+  const lc = await new GenericContainer(
+    'public.ecr.aws/localstack/localstack:3',
+  )
+    .withName(`localstack-${Math.random().toString(16).slice(2, 8)}`)
+    .withExposedPorts(4566)
+    .withWaitStrategy(Wait.forLogMessage('Ready.'))
+    .start()
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ;(global as any).__localstack__ = lc
+
+  process.env.SQS_ENDPOINT = `http://${lc.getHost()}:${lc.getMappedPort(4566)}`
```

Comment:
~I see two occurrences of this port, should it come from a config or at least be in a variable?